backend/gateway/mux/util_test.go

```diff
@@ -51,3 +51,17 @@ func TestRequestHeadersFromrResponseWriterMessed(t *testing.T) {
 	ret := requestHeadersFromResponseWriter(m)
 	assert.Equal(t, "", ret.Get("foo"))
 }
+
+func TestGetCookieValue(t *testing.T) {
+	cookies := []string{
+		"foo=bar;baz=bang",
+		"baz=bloop",
+	}
+	res, err := GetCookieValue(cookies, "foo")
+	assert.NoError(t, err)
+	assert.Equal(t, "bar", res)
+
+	res, err = GetCookieValue(cookies, "baz")
```

Comment:
should we add a test that when we ask for something that doesn't exist nothing comes back?